haystack/components/generators/chat/openai.py

```diff
@@ -13,9 +13,6 @@
 logger = logging.getLogger(__name__)
 
 
-API_BASE_URL = "https://api.openai.com/v1"
```

Comment:
Keeping this now that we live in the world of multiple providers just didn't make sense. I didn't see use for it